### PR TITLE
Add all the packages for moving over the rails SCL

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

**Comment by pcreech** [comment]
@zjhuntin The following packages aren't in the package_manifest:

* rubygem-activepack
* rubygem-activeview
* rubygem-concurrent-rails

**Comment by zjhuntin** [comment]
@pcreech updated :)

File: packages/foreman/rubygem-actioncable/rubygem-actioncable.spec [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+%exclude %{gem_cache}
+%{gem_spec}
+
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/CHANGELOG.md
+%doc %{gem_instdir}/README.md
+
+%changelog
+* Wed Aug 22 2018 Eric D. Helms <ericdhelms@gmail.com> 5.2.1-1
```
Comment (ehelms) [inline packages/foreman/rubygem-actioncable/rubygem-actioncable.spec]: I'd vote that we bump the release on each package so we have a changelog entry indicating the switch to tfm-

**Comment by ehelms** [comment]
Should this include the update to tito.props so the packages are whitelisted?

File: packages/foreman/rubygem-actioncable/rubygem-actioncable.spec (hunk 1 above)
Comment (zjhuntin) [reply packages/foreman/rubygem-actioncable/rubygem-actioncable.spec]: Will do!

**Comment by zjhuntin** [comment]
@ehelms I think so, I'll add that too

**Comment by ekohl** [comment]
In this repo we have a slightly different gem2rpm template and adds markers so `bump_rpm.sh` can automatically update dependencies when bumping. Does it make sense to regenerate the specs to include those?

**Comment by zjhuntin** [comment]
@ekohl it sounds like a plan, how do I go about doing that?

**Comment by ekohl** [comment]
You can use `add_gem_package.sh` and it'll regenerate the file based on the template. It also does some effort to keep the changelog. Does reset the release back to `1` so that may need some manual fixing. I'd also recommend to (at least temporarily) modify the template to add the Obsoletes line.

**Comment by zjhuntin** [comment]
@ekohl Updated with all of the spec files regenerated!

**Comment by pcreech** [comment]
@zjhuntin looks like the job failed out with an inability to parse the spec files

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
Closing for now in favor of https://github.com/theforeman/foreman-packaging/pull/4578 & https://github.com/theforeman/foreman-packaging/pull/4579 to speed up testing.

**Pull request closed without merging**